api/controllers/user.js

```diff
@@ -447,19 +447,18 @@ async function storePassword(req, res) {
       userId: userid,
       status: false,
     }).exec();
-    if (!resetPassword) {
+    const expireTime = moment.utc(resetPassword.resetPasswordExpires).isBefore(moment());
```

Comment:
Will this work properly? If `resetPassword` is undefined then we'll most likely get a TypeError if we try to access `resetPasswordExpires`.

Reply:
I think you are right! . I delete that if by mistake... I mean the if should persist but with a different message.